server/src-lib/Hasura/GraphQL/Execute/RemoteJoins.hs

```diff
@@ -595,15 +602,16 @@ extractRemoteRelArguments remoteSchemaMap hasuraJson rels =
                                  (rrRemoteField remoteRel)))
                            remoteSchemaMap of
                       Just remoteSchemaCtx ->
-                        pure (remoteRel, (rscInfo remoteSchemaCtx), rows)
+                        -- TODO the first two parts of this 3-tuple are just discarded immediately below; are we missing functionality here?
+                        pure (remoteRel, rscInfo remoteSchemaCtx, rows)
```

Comment:
We could make this:

                          Just _remoteSchemaCtx ->
                             pure rows

But then it's not clear if we actually need/want to check that the remote schema info is in the map at all. It seems like maybe this function is unfinished?